From 2f2469b859ddc78f989e365aaffccbabc8bd465f Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Wed, 3 Apr 2024 10:09:32 -0400
Subject: [PATCH] Don't update icon cache

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 99c4228..c1c1d4a 100644
--- a/meson.build
+++ b/meson.build
@@ -348,7 +348,7 @@ endif
 
 gnome.post_install(
   glib_compile_schemas: true,
-  gtk_update_icon_cache: true,
+  gtk_update_icon_cache: false,
 )
 
 configure_file(
