From 7a9f989f9ac74b6414982af8ef27848394a8edf8 Mon Sep 17 00:00:00 2001
From: Bruce Leidl <bruce@subgraph.com>
Date: Mon, 18 Sep 2023 15:28:14 -0400
Subject: [PATCH] fix init configure

---
 build/moz.configure/init.configure | 31 +++++++++++++++---------------
 1 file changed, 16 insertions(+), 15 deletions(-)

diff --git a/build/moz.configure/init.configure b/build/moz.configure/init.configure
index 2ac40d3d39..dc8a605d06 100644
--- a/build/moz.configure/init.configure
+++ b/build/moz.configure/init.configure
@@ -650,23 +650,24 @@ def help_host_target(help, host, target):
 
 def config_sub(shell, triplet):
     config_sub = os.path.join(os.path.dirname(__file__), "..", "autoconf", "config.sub")
+    return triplet
     # Config.sub doesn't like the *-windows-msvc/*-windows-gnu triplets, so
     # munge those before and after calling config.sub.
-    suffix = None
-    munging = {
-        "-windows-msvc": "-mingw32",
-        "-windows-gnu": "-mingw32",
-    }
-    for check_suffix, replacement in munging.items():
-        if triplet.endswith(check_suffix):
-            suffix = check_suffix
-            triplet = triplet[: -len(suffix)] + replacement
-            break
-    result = check_cmd_output(shell, config_sub, triplet).strip()
-    if suffix:
-        assert result.endswith(replacement)
-        result = result[: -len(replacement)] + suffix
-    return result
+    #suffix = None
+    #munging = {
+    #    "-windows-msvc": "-mingw32",
+    #    "-windows-gnu": "-mingw32",
+    #}
+    #for check_suffix, replacement in munging.items():
+    #    if triplet.endswith(check_suffix):
+    #        suffix = check_suffix
+    #        triplet = triplet[: -len(suffix)] + replacement
+    #        break
+    #result = check_cmd_output(shell, config_sub, triplet).strip()
+    #if suffix:
+    #    assert result.endswith(replacement)
+    #    result = result[: -len(replacement)] + suffix
+    #return result
 
 
 @depends("--host", shell)
